internal/collector/daemonset.go

```diff
@@ -25,7 +25,7 @@ func (f *Factory) ReconcileDaemonset(er record.EventRecorder, k8sClient client.C
 	var receiverInputs []string
 	for _, input := range f.ForwarderSpec.Inputs {
 		if logging.IsHttpReceiver(&input) || logging.IsSyslogReceiver(&input) {
-			receiverInputs = append(receiverInputs, input.Name)
+			receiverInputs = append(receiverInputs, f.ResourceNames.GenerateInputServiceName(input.Name))
```

Comment:
I have another question further down but not certain why this change is needed?

Reply:
See above comments about receiver input names and services: https://github.com/openshift/cluster-logging-operator/pull/2304#discussion_r1459489323